data-and-functions-1/src/functions/getProductById.js

```diff
@@ -0,0 +1,13 @@
+import DATA from '../DATA'
```

Comment:
You don't need to `import` the data, since the test will pass it in as the first parameter.